pages/shared/[folderId].tsx

```diff
@@ -0,0 +1,73 @@
+import { useGetFolder } from "@/src/folder/data-access-folder";
+import { Layout } from "@/src/sharing/feature-layout";
+import { SharedLayout } from "@/src/page-layout/SharedLayout";
+import { CardList } from "@/src/link/ui-card-list";
+import { FolderInfo } from "@/src/folder/ui-folder-info";
+import { ReadOnlyCard } from "@/src/link/ui-read-only-card";
+import { SearchBar } from "@/src/link/ui-search-bar";
+import { useSearchLink } from "@/src/link/util-search-link/useSearchLink";
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+interface UserData {
+  id: number;
+  name: string;
+  email: string;
+  image_source: string;
+}
+
+const SharedPage = () => {
+  const { data } = useGetFolder();
+  const { profileImage, ownerName, folderName, links } = data || {};
+  const { searchValue, handleChange, handleCloseClick, result } =
+    useSearchLink(links);
+
+  const [userData, setUserData] = useState<UserData | null>(null);
+
+  useEffect(() => {
+    const fetchUserData = async () => {
+      try {
+        const res = await axios.get<UserData>(
```

Comment:
여기도 UserData[] 로 타입 변경해주세요
```suggestion
        const res = await axios.get<UserData[]>(
```